pkg/machines/components/networks/networkOverviewTab.jsx

```diff
@@ -48,82 +52,109 @@ const DHCPHost = (host, index, family, idPrefix) => {
     </React.Fragment>);
 };
 
-export const NetworkOverviewTab = ({ network }) => {
-    const idPrefix = `${networkId(network.name, network.connectionName)}`;
+export class NetworkOverviewTab extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.onAutostartChanged = this.onAutostartChanged.bind(this);
+    }
+
+    onAutostartChanged() {
+        const { dispatch, network } = this.props;
+        const autostart = !network.autostart;
+
+        dispatch(changeNetworkAutostart({ network, autostart }))
```

Comment:
I don't think dispatch is needed here.
In general dispatch is used to wrap methods that are calling redux actions which are handled by redux reducers.
All other method do not need dispatch wrapper.